src/App.js

```diff
@@ -20,6 +21,7 @@ function App() {
           <Route path="/product/:productId" element={<SingleProductPage />} />
           <Route path="/product" element={<Product />} />
           <Route path="/cart" element={<Cart />} />
+          <Route path="/mockman" element={<Mockman />} />
         </Routes>
       </BrowserRouter>
```

Comment:
You should have your <BrowserRouter> in your index.js, here you should just have Routes.